docs/soyousegfaulted.md

```diff
@@ -0,0 +1,48 @@
+# So you got a segfault 
+
+While time and effort have been put into ensuring a bug-free experience, some edge cases may still arise that result in a segfault. 
```

Comment:
I generally try to avoid writing defensively like this in technical docs. The user normally just cares about finding out how to troubleshoot. Some proposals:

* rename file to `segfault.md`
* use title "How to troubleshoot a segfault"
* first sentence explains the purpose of the document, something like "This document explains the steps to report an issue related to segfaults".
* create GitHub issue template for reporting a segfault. This could replace the "Info to be provided" header. Link to this document from the template.